extensions/http-curl/tests/unit/InvokeHTTPTests.cpp

```diff
@@ -307,6 +315,6 @@ TEST_CASE("HTTPTestsPostNoResourceClaim", "[httptest1]") {
     REQUIRE(provEventRecord->getComponentType() == processor->getName());
```

Comment:
Correct me if I am wrong but `record = plan->getCurrentFlowFile()` has been done twice

Reply:
Removed unused `getCurrentFlowFile` calls in 50926ed98c23a14b8f7113563aeb887aacb6678d